pkg/tfbridge/diff.go

```diff
@@ -111,10 +111,15 @@ func visitPropertyValue(name, path string, v resource.PropertyValue, tfs shim.Sc
 					}
 				}
 
-				ti = strconv.FormatInt(int64(tfs.SetHash(ev)), 10)
-				if containsComputedValues(e) {
-					// TF adds a '~' prefix to the hash code for any set element that contains computed values.
-					ti = "~" + ti
+				if !e.IsComputed() {
+					// We cannot compute the hash for computed values because they are represented by the UnknownVariableValue
+					// sentinel string, which may not be a legal value of the corresponding schema type, and SetHash does not
+					// account for computed values.
```

Comment:
it would be good to expand on what will be present in the instance diff from TF in this case (experimentally, only a diff on the count property, which we will check for elsewhere).